### ISSUE TITLE
Checkbox: ARIA attribute "aria-describedby" is getting attached without any reference value in 5.x

### ISSUE THREAD

**Issue by kmrrakesh** [OfficeDev/office-ui-fabric-react#4944]
Bug Report
Describe the issue :
ARIA attribute, "aria-describedby" for checkbox is getting attached, without any reference value

It is happening in the latest fabric ui, for all chekbox with/without associated label, and run keros to replicate it.
It complains as "ARIA attributes must conform to valid values"

Actual behavior:
Checkbox is associating an attribute as "aria-describedby" without value

Expected behavior:
Checkbox attribute "aria-describedby", should have relevant value

**Comment by micahgodbolt** [comment]
What version of Fabric are you using, and can you provide a reporable example? I don't see any issue with our current release.

**Comment by arunarokk** [comment]
@micahgodbolt , this bug is repro on the version 5.110, its resolved in current version.

**Comment by micahgodbolt** [comment]
So the problem is that this fix needs to be backported to 5.x.

What version of Fabric are you using? Are you able to upgrade to 6.x? If not, would you be able to create a PR to backport the changes to 5.x? I'd be happy to review/approve the PR.

**Comment by arunarokk** [comment]
we will be upgrading to 6.x very soon.
